src/migrations/20240520201321-remove-unused-objective-tables.js

```diff
@@ -0,0 +1,308 @@
+const { SUPPORT_TYPES } = require('@ttahub/common');
+const { prepMigration } = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+      await queryInterface.removeColumn('Objectives', 'supportType', { transaction });
+      await queryInterface.dropTable('ZALObjectiveTemplateTopics', { transaction });
+      await queryInterface.dropTable('ZALObjectiveTemplateFiles', { transaction });
+      await queryInterface.dropTable('ZALObjectiveTemplateResources', { transaction });
+      await queryInterface.dropTable('ZALObjectiveCourses', { transaction });
+      await queryInterface.dropTable('ZALObjectiveTopics', { transaction });
+      await queryInterface.dropTable('ZALObjectiveFiles', { transaction });
+      await queryInterface.dropTable('ZALObjectiveResources', { transaction });
+      await queryInterface.dropTable('ObjectiveTemplateTopics', { transaction });
+      await queryInterface.dropTable('ObjectiveTemplateFiles', { transaction });
+      await queryInterface.dropTable('ObjectiveTemplateResources', { transaction });
+      await queryInterface.dropTable('ObjectiveCourses', { transaction });
+      await queryInterface.dropTable('ObjectiveTopics', { transaction });
+      await queryInterface.dropTable('ObjectiveFiles', { transaction });
+      await queryInterface.dropTable('ObjectiveResources', { transaction });
```

Comment:
const { prepMigration, removeTables } = require('../lib/migration');

```suggestion
     await removeTables(queryInterface, transaction , [
         'ObjectiveTemplateTopics',
         'ObjectiveTemplateFiles',
         'ObjectiveTemplateResources',
         'ObjectiveCourses',
         'ObjectiveTopics',
         'ObjectiveFiles',
         'ObjectiveResources',
     ]);
```